airbyte-integrations/connectors-performance/destination-harness/src/main/resources/catalogs/destination-snowflake/10m_catalog.json

```diff
@@ -71,7 +71,7 @@
       "sync_mode": "full_refresh",
       "primary_key": [["id"]],
       "cursor_field": ["updated_at"],
-      "destination_sync_mode": "append"
+      "destination_sync_mode": "overwrite"
```

Comment:
why is this changing?

Reply:
I changed primarily to get a clean slate upon each run. The alternative is to:
* add some uuid to the stream_name for each run
* and tear down the streams after the sync

This was a quick and dirty solution

Reply:
Clean slate in the destination?

Reply:
Yep, it's so when we do a 

```SQL
SELECT count(*) from schema.stream
```

it will not give us more records due to having records from a previous stream. I mean it's not difficult to add a `WHERE timestamp > sync_time` but thought it would be nice to have a clean slate between runs

Reply:
understand. this makes sense!